pkg/i18n/english.go

```diff
@@ -1166,51 +1175,55 @@ func EnglishTranslationSet() TranslationSet {
-		ResetCherryPick:            "Reset cherry-picked (copied) commits selection",
-		NextTab:                    "Next tab",
-		PrevTab:                    "Previous tab",
-		CantUndoWhileRebasing:      "Can't undo while rebasing",
-		CantRedoWhileRebasing:      "Can't redo while rebasing",
-		MustStashWarning:           "Pulling a patch out into the index requires stashing and unstashing your changes. If something goes wrong, you'll be able to access your files from the stash. Continue?",
-		MustStashTitle:             "Must stash",
-		ConfirmationTitle:          "Confirmation panel",
-		PrevPage:                   "Previous page",
-		NextPage:                   "Next page",
-		GotoTop:                    "Scroll to top",
-		GotoBottom:                 "Scroll to bottom",
-		FilteringBy:                "Filtering by",
-		ResetInParentheses:         "(Reset)",
-		OpenFilteringMenu:          "View filter-by-path options",
-		FilterBy:                   "Filter by",
-		ExitFilterMode:             "Stop filtering by path",
-		FilterPathOption:           "Enter path to filter by",
-		EnterFileName:              "Enter path:",
-		FilteringMenuTitle:         "Filtering",
-		MustExitFilterModeTitle:    "Command not available",
-		MustExitFilterModePrompt:   "Command not available in filter-by-path mode. Exit filter-by-path mode?",
-		Diff:                       "Diff",
-		EnterRefToDiff:             "Enter ref to diff",
-		EnterRefName:               "Enter ref:",
-		ExitDiffMode:               "Exit diff mode",
-		DiffingMenuTitle:           "Diffing",
-		SwapDiff:                   "Reverse diff direction",
-		OpenDiffingMenu:            "Open diff menu",
+		IgnoreTracked:                    "Ignore tracked file",
+		IgnoreTrackedPrompt:              "Are you sure you want to ignore a tracked file?",
+		ExcludeTracked:                   "Exclude tracked file",
+		ExcludeTrackedPrompt:             "Are you sure you want to exclude a tracked file?",
+		ViewResetToUpstreamOptions:       "View upstream reset options",
+		NextScreenMode:                   "Next screen mode (normal/half/fullscreen)",
+		PrevScreenMode:                   "Prev screen mode",
+		StartSearch:                      "Search the current view by text",
+		StartFilter:                      "Filter the current view by text",
+		Panel:                            "Panel",
+		KeybindingsLegend:                "Legend: `<c-b>` means ctrl+b, `<a-b>` means alt+b, `B` means shift+b",
+		RenameBranch:                     "Rename branch",
+		BranchUpstreamOptionsTitle:       "Upstream options",
+		ViewBranchUpstreamOptionsTooltip: "View options relating to the branch's upstream e.g. setting/unsetting the upstream and resetting to the upstream",
+		UpstreamNotStoredLocallyError:    "Cannot reset to upstream branch because it is not stored locally",
+		UpstreamNotSetError:              "You cannot reset to selected branch upstream because it has not been set!",
```

Comment:
```suggestion
		UpstreamNotSetError:              "The selected branch has no upstream",
```
I'd keep it generic so we can use it in other places too without needing a new translation string every time

Reply:
I took the liberty of applying this change (see 30fc1c121bb2) so that we can get this merged.

Reply:
Sounds great, thanks!